src/app/public/modules/popover/popover.directive.ts

```diff
@@ -73,7 +73,6 @@ export class SkyPopoverDirective implements OnInit, OnDestroy {
 
   /**
    * Specifies the user action that displays the popover.
-   * @deprecated The trigger type `mouseenter` will be removed in the next major version.
```

Comment:
Why did we just remove the `deprecated` annotation here but leave the property?

Reply:
Disregard. I see this was a merge from master.

Reply:
Todd wanted us to revert the deprecation for hover-activated popovers